graders/c/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:24.04
-
```

Comment:
Why remove the newline here and not from `graders/java/Dockerfile`?

Reply:
Fixed in [bb90ef0](https://github.com/PrairieLearn/PrairieLearn/pull/11696/commits/bb90ef019608f09f40a5024fdf453c3e74d90c3e).